govdelivery/xml_response_parsers.py

```diff
@@ -4,8 +4,9 @@
 def topic_xml_as_dict(unparsed_xml, only_listed=False):
     root = ET.fromstring(unparsed_xml)
     code_map = {}
+
     for topic_tag in root.findall('topic'):
-        if topic_tag.find('visibility').text == 'Unlisted':
+        if topic_tag.find('visibility').text == 'Unlisted' and only_listed:
```

Comment:
Is this a bug fix?

Reply:
Yes, it seems that this previously did not work as intended.